experimental/cloudstrapper/playbooks/roles/mantle-base/tasks/main.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
minor nit: remove ?

Reply:
Removed a number of Mantle related code which is not required anymore. However, keeping just one task, which is the creation of the home directory for experimental where all the runtime artifacts are stored.